tasks/install-from-source.yml

```diff
@@ -1,27 +1,39 @@
 ---
+- name: Define git_install_from_source_dependencies.
+  set_fact:
+    git_install_from_source_dependencies: "{{ __git_install_from_source_dependencies | list }}"
+  when:
+  - git_install_from_source_dependencies is not defined
```

Comment:
This condition should just be `git_install_from_source_dependencies is not defined` (without the second condition)—shouldn't `__git_install_from_source_dependencies` always be defined?